ekip/everykid/templates/about.html

```diff
@@ -141,23 +123,23 @@
   <div class="contact">
     <div class="contact-content">
 
-      <img src="{% static "img/about-turtle.jpg" %}" alt="Kids learning about turtles." />
-      <h2>Media</h2>
+      <img src="{% static "img/about-turtle.jpg" %}" alt="{% trans "Kids learning about turtles." %}" />
+      <h2>{% trans "Media" %}</h2>
 
-      <p>Here's an <a href="{% static "media/info-kit-en.zip" %}">information kit</a> in English (ZIP file, 67.1 MB)</a>.</p>
-      <p>Here's an <a href="{% static "media/info-kit-es.zip" %}">information kit</a> in Spanish (ZIP file, 5.4 MB)</a>.</p>
-      <p>Please don't hesitate to <a href="mailto:interior_press@ios.doi.gov">email the press office</a> if you have additional questions.</p>
+      <p>{% trans "Here's an" %} <a href="{% static "media/info-kit-en.zip" %}">{% trans "information kit" %}</a> {% trans "in English" %} (ZIP file, 67.1 MB)</a>.</p>
+      <p>{% trans "Here's an" %} <a href="{% static "media/info-kit-es.zip" %}">{% trans "information kit" %}</a> {% trans "in Spanish" %} (ZIP file, 5.4 MB)</a>.</p>
+      <p>{% trans "Please don't hesitate to" %} <a href="mailto:interior_press@ios.doi.gov">{% trans "email the press office" %}</a> {% trans "if you have additional questions." %}</p>
       
-      <h2>Tools</h2>
+      <h2>{% trans "Tools" %}</h2>
 
-      <p>Here's a <a href="{% static "media/tool-kit-2.zip" %}">set of resources</a> (ZIP file, 15.4 MB) to help local officials, youth leaders and others plan an outdoor outing with 4th graders.</p>
-      <p>In partnership with the Every Kid in a Park Program, Transforming Youth Outdoors <a href="https://tyo.pathgather.com/l/login" target="_blank">has created two step-by-step learning paths</a> that give classroom teachers and outdoor educators tools to learn more about the Every Kid program and plan a successful field trip. These resources provide information about federal lands and waters, tools to find nearby locations, and information on planning and executing an Every Kid in a Park field trip.</p>
+      <p>{% trans "Here's a" %} <a href="{% static "media/tool-kit-2.zip" %}">{% trans "set of resources" %}</a> (ZIP file, 15.4 MB) {% trans "to help local officials, youth leaders and others plan an outdoor outing with 4th graders." %}</p>
+      <p>{% trans "In partnership with the Every Kid in a Park Program, Transforming Youth Outdoors" %} <a href="https://tyo.pathgather.com/l/login" target="_blank">{% trans "has created two step-by-step learning paths" %}</a> {% trans "that give classroom teachers and outdoor educators tools to learn more about the Every Kid program and plan a successful field trip. These resources provide information about federal lands and waters, tools to find nearby locations, and information on planning and executing an Every Kid in a Park field trip." %}</p>
```

Comment:
I would recommend wrapping long text lines such as these, so that it's a bit easier to read.